nixos/modules/services/monitoring/scrutiny-collector.nix

```
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+in
+{
+  options.services.scrutiny-collector = {
+    enable = mkEnableOption (mdDoc "scrutiny-collector");
+
+    package = mkPackageOption pkgs "scrutiny-collector" { };
+
+    config-path = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = mdDoc ''
+        Specify the path to the devices file
+      '';
+    };
+
+    configuration = mkOption {
+      type = types.nullOr (pkgs.formats.yaml { }).type;
+      default = null;
+      description = lib.mdDoc ''
+        Specify the configuration for the Scrutiny collector in Nix.
+      '';
+    };
+
+    api-endpoint = mkOption {
+      type = types.str;
+      default = "http://localhost:8080";
+      description = mdDoc ''
+        The api server endpoint
+      '';
+    };
+
+    log-file = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = mdDoc ''
+        Path to file for logging. Leave empty to use STDOUT
+      '';
+    };
+
+    debug = mkEnableOption "debug logging";
+
+    host-id = mkOption {
+      type = types.str;
+      default = config.networking.hostName;
+      defaultText = lib.literalExpression "config.networking.hostName";
+      description = mdDoc ''
+        Host identifier/label, used for grouping devices
+      '';
+    };
+
+    calendar = mkOption {
+      type = types.str;
+      default = "*-*-* 00:00:00";
+      description = mdDoc ''
+        Configured when to run the service systemd unit (DayOfWeek Year-Month-Day Hour:Minute:Second).
+      '';
+    };
+
+    user = mkOption {
+      type = types.str;
+      default = "scrutiny-collector";
+      description = mdDoc ''
+        User under which the scrutiny collector service runs.
+      '';
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "scrutiny-collector";
+      description = mdDoc ''
+        Group under which the scrutiny collector service runs.
+      '';
```

```suggestion
      description = " Group under which the scrutiny collector service runs.";
```
same should go for other single line descriptions imo.